javascript/utils/geoUtils.js

```diff
@@ -28,18 +28,18 @@ export function makeFeatureCollection(features = []) {
   return featureCollection(features);
 }
 
-export function addToFeatureCollection(featureCollection, feature) {
-  const shallowFeatureCollection = Object.assign({}, featureCollection);
-  shallowFeatureCollection.features.push(feature);
+export function addToFeatureCollection(newFeatureCollection, newFeature) {
+  const shallowFeatureCollection = Object.assign({}, newFeatureCollection);
+  shallowFeatureCollection.features.push(newFeature);
   return featureCollection;
```

Comment:
```suggestion
  return newFeatureCollection;
```

BTW i'm not sure the idea behind `Object.assign({},` eg why we don't just do

```js
newFeatureCollection.features.push(newFeature);
return newFeatureCollection;
```

Reply:
Thanks, will fix. Reason for Object.assign is to not mutate the input object, which is bad practice.

Reply:
It mutates `newFeatureCollection.features` array, but not `newFeatureCollection` object itself, so i don't think assign is needed.